common/crypto_utils/avalon_crypto_utils/signature.py

```diff
@@ -365,6 +362,91 @@ def verify_signature(self, input_json, verification_key):
         else:
             return SignatureStatus.INVALID_SIGNATURE_FORMAT
 
+    def _verify_wo_verification_key_signature(self, wo_response,
+                                              wo_verification_key,
+                                              requester_nonce):
+        """
+        Function to verify the work order response signature
+        Parameters:
+            @param wo_response - dictionary contains work order response
+            as per Trusted Compute EEA API 6.1.2 Work Order Result Payload
+            @param wo_verification_key - ECDSA/SECP256K1 public key used
+            to verify work order verification key signature.
+            @param requester_nonce - requester generated nonce passed in work
+            order request. Required in 2 step verification.
+        Returns enum type SignatureStatus
+        """
+        if requester_nonce is None:
+            logger.error("Missing requester_nonce argument")
+            return SignatureStatus.FAILED
+
+        decoded_v_key = crypto.base64_to_byte_array(
+                wo_response["extVerificationKey"])
+        v_key = crypto.byte_array_to_string(decoded_v_key)
```

Comment:
We can use `extVerificationKey` directly since it's a base64 encoded string. We don't need decoded_v_key and v_key since there is no need to convert
base64 string -> bytes -> string

Reply:
It was needed because verification_key is passed in base64 format. We need plain key with begin and end to verify signature.